pytorch_lightning/callbacks/early_stopping.py

```diff
@@ -88,7 +88,7 @@ class EarlyStopping(Callback):
 
     def __init__(
         self,
-        monitor: str = 'early_stop_on',
+        monitor: str = Optional[None],
```

Comment:
this is invalid python. Optional is a typehint, which needs to be around the str, like this
```suggestion
        monitor: Optional[str] = None,
```

Reply:
Oh alright.